API.yaml

```diff
@@ -1867,6 +1867,9 @@ components:
             optional if this entry does not have a parent.
           type: integer
           format: int32
+        hasRowModel:
```

Comment:
This one is only applicable for time graphs and should be added to TimeGraphEntry instead.

Reply:
Actually, TmfTreeDataModel has this now too, because some tree entries for XY charts may have no data, but the name is misleading. Here it should be hasData, to make the difference with entries that have no data (so no legend to assign to them)

Reply:
Then a generic attribute with generic name makes sense.